web/bfex/components/scraper/scraper_factory.py

```diff
@@ -5,9 +5,21 @@
 
 
 class ScraperFactory(object):
+        """ Returns a scrapper made for the URL you feed it
+        """
 
     @staticmethod
     def create_scraper(url, type):
+        """ Each type of website will need a different scrapper to go through things
```

Comment:
scrapper

Reply:
I'm just going to ignore the other scrapper errors, if any. Since we changed the names of the classes and files, documentation should match.